debian/changelog

```diff
@@ -8,24 +8,27 @@ ubuntu-advantage-tools (27.0.2) impish; urgency=medium
     - apt-hook: refactor json hook messaging to be dry
     - tests: fix subp ls error case for powerpc builds
     - jenkinsfile: add --resolve-alternatives for trusty builds
+    - amend changelog: add omitted apt-hook message for 27.0.1 stanza
 
  -- Chad Smith <chad.smith@canonical.com>  Fri, 07 May 2021 11:58:03 -0600
 
 ubuntu-advantage-tools (27.0.1) impish; urgency=medium
 
   * Add .gitignore and cleanup ignored directory .pytest_cache
+  * apt-hook: mitigate failures with true
 
- -- Chad Smith <chad.smith@canonical.com>  Fri, 07 May 2021 11:55:07 -0600
+ -- Chad Smith <chad.smith@canonical.com>  Wed, 28 Apr 2021 13:55:28 -0600
 
 ubuntu-advantage-tools (27.0) impish; urgency=medium
 
   * New upstream release 27.0:
-    - apt-hook: mitigate failures with true
+    - [redacted: actually landed in 27.0.1] apt-hook: mitigate failures with
```

Comment:
Sorry if I am missing something here, but do we need to have those updates on the changelog ? Can't we just remove keep the correct line on release 27.0.1 and remove the other ones ?

Reply:
@lucasmoura since we already released a d/changelog in impish incorrectly published this line, Robie thought it was more correct and less confusing to others who may have seen the impush release changelog to avoid a complete redaction of a line item. It was a point raised in SRU that we probably shouldn't be magically manipulating old d/changelog entries if we can avoid it.